src/impl/realm_coordinator.cpp

```diff
@@ -860,6 +864,9 @@ void RealmCoordinator::process_available_async(Realm& realm)
     if (notifiers.empty())
         return;
 
+    if (realm.m_binding_context)
+        realm.m_binding_context->will_send_notifications();
```

Comment:
The error case right below this is missing a call to did_send_notifications()

Reply:
Fixed